From 5fad96254cf20fc6d473037b48c50031cd7e8b45 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Fri, 3 Apr 2020 18:35:21 +0200
Subject: [PATCH] Drop tags and ctags targets

meson 0.43 started providing a built-in ctags target which
conflicts with the hand-rolled one here. There shouldn't be
much difference, so let's drop ours.

Fixes #22.
---
 meson.build | 22 ----------------------
 1 file changed, 22 deletions(-)

diff --git a/meson.build b/meson.build
index 5fc51d2..72ac786 100644
--- a/meson.build
+++ b/meson.build
@@ -79,25 +79,3 @@ subdir('lib')
 subdir('tool')
 subdir('vim')
 subdir('bash-completion')
-
-############################################################
-
-git = find_program('git', required : false)
-
-if git.found()
-        all_files = run_command(
-                git,
-                ['--git-dir=@0@/.git'.format(meson.current_source_dir()),
-                 'ls-files',
-                 ':/*.[ch]'])
-        all_files = files(all_files.stdout().split())
-
-        custom_target(
-                'tags',
-                output : 'tags',
-                command : ['env', 'etags', '-o', '@0@/TAGS'.format(meson.current_source_dir())] + all_files)
-        custom_target(
-                'ctags',
-                output : 'ctags',
-                command : ['env', 'ctags', '-o', '@0@/tags'.format(meson.current_source_dir())] + all_files)
-endif
